Describe how to instantiate motivating use cases (#47)
tfpauly/privacy-proxy: draft-private-access-tokens.md

Origins almost exclusively use client IP addresses for this purpose.

## Client Linking

> An origin wishes to link multiple accesses from a client. The server does not

Reviewer: Can we mention this in privacy considerations, and say that a Client replaying tokens (and origins accepting that) would allow this happen again.

Author: I'll address this in a followup PR.

Before forwarding the Client's request to the Issuer, the Mediator adds headers
listing both the ANON_CLIENT_ID as "Anonymous-Client-ID", and the ANON_ORIGIN_ID_PRIME as

> "Anonymous-Origin-ID". The Mediator MAY add additional context information about Clients,
> such as their geographic region encapsulated in a "Sec-CH-Geohash" header {{!I-D.ietf-geohash-hint}}.

Reviewer: I'd leave this out for now from this part and only mention it below.

```suggestion
"Anonymous-Origin-ID". The mediator MAY also add additional context information, but MUST
NOT add information that will uniquely identify a client.
```

its budget. Failure to present a token can be interpreted as a signal that the client's token
budget was exceeded.

## Client Geo-Location {#implement-geolocation}

To instantiate this use case, the Issuer has an issuing key pair per geographic region, i.e.,

> each region has a unique policy key. The Mediator forwards information about the Client's

Reviewer: I don't know if the mediator would be sending the geohash to the Issuer here—instead, it should have a URI to look up on an issuer what key_id should be expected for each region, so it can verify the key is correct.

Author: Ah, that's much better!
